### DISCUSSION TITLE
i cant login in vercel

### DISCUSSION THREAD

**Discussion by tboy007** [vercel/community#4421] [Help]
### Summary

We are unable to complete your signup. Please read [this guide](https://vercel.com/guides/why-can-i-not-signup) for further details.

this show like this , how can i fix this

### Steps to Reproduce

We are unable to complete your signup. Please read [this guide](https://vercel.com/guides/why-can-i-not-signup) for further details.

this show like this , how can i fix this

**Comment by amyegan** [comment]
You may have received an email with instructions on how to solve the problem. Please check your spam or junk folder in case the message was filtered out of your main email inbox.

**Comment by tboy007** [comment]
yeah i have but i click this this show time out , or you mean i only pro
version only for used ?

ในวันที่ อ. 10 ต.ค. 2023 เวลา 20:56 Amy ***@***.***> เขียนว่า:

**Comment by tboy007** [comment]
this show

Authentication Failed

It looks like you may have clicked on an invalid email verification link.
Please close this window and try authenticating again.

ในวันที่ อ. 10 ต.ค. 2023 เวลา 23:04 tonmai tonmAI ***@***.***>
เขียนว่า:
